scripts/check-events

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/ruby
+# coding: utf-8
+
+require 'find'
```

Comment:
Does this need to be added to `scripts/Gemfile`?

Reply:
I have no idea. It's part of the base ruby install on my system, so I _think_ not.